### PR TITLE
Fix broken links to other markdown pages

### PR DESCRIPTION
**Pull request by niloc132**
Does not fix release notes, javadocs, nor any external links.

### CODE REVIEW

File: src/main/markdown/gettingstarted-v2.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Getting Started
 ===
 
-Starting with 2.11.0 of the GWT SDK the [webAppCreator](gettingstarted-outdated.html) is deprecated. Instead, we will use 
+Starting with 2.11.0 of the GWT SDK the [webAppCreator](gettingstarted.html) is deprecated. Instead, we will use
```
Comment (niloc132) [inline src/main/markdown/gettingstarted-v2.md:4]: note that there is no `-outdated` file - perhaps there was at one point, but it never merged.

File: src/main/markdown/gettingstarted.md [hunk 2]
```diff
@@ -159,4 +159,4 @@ set up Eclipse to use the GWT SDK:
 [Set up Eclipse](usingeclipse.html)
 
 If you are going to stick with the command line, check out Speed Tracer     and then
-head over to [Build a Sample GWT App](doc/latest/tutorial/gettingstarted-v2.html).
+head over to [Build a Sample GWT App](doc/latest/tutorial/gettingstarted.html).
```
Comment (niloc132) [inline src/main/markdown/gettingstarted.md:162]: This is the "outdated" file, so I don't feel bad for the moment pointing to an old getting started doc.

That said, we don't have a doc/latest/tutorial/gettingstarted-v2 yet, just the original.

**Review by niloc132 — COMMENTED** [review]
This PR aims to help get us to the point of having all working links on the site, so that we can set up a PR check that validates internal links and nightly job for reporting broken external links.

**Review by FrankHossfeld — APPROVED** [review]

File: src/main/markdown/gettingstarted-v2.md (hunk 1 above)
Comment (FrankHossfeld) [reply src/main/markdown/gettingstarted-v2.md:4]: might be an refactoring issue.

File: src/main/markdown/gettingstarted.md (hunk 2 above)
Comment (FrankHossfeld) [reply src/main/markdown/gettingstarted.md:162]: There is an unmerged branch 'getting-started-v2' which is containing the file

**Pull request merged**